File: app/src/main/java/com/cheatdatabase/listitems/CheatListItem.java
package com.cheatdatabase.listitems;

import com.cheatdatabase.model.Cheat;

public class CheatListItem extends ListItem {

    private Cheat cheat;

    public Cheat getCheat() {
        return cheat;
    }

    public void setCheat(Cheat cheat) {
        this.cheat = cheat;
    }

    @Override
    public int getType() {
        return ListItem.TYPE_CHEAT;
    }

    @Override
    public String getTitle() {
        return cheat.getCheatTitle().substring(0, 1);
    }
}

File: app/src/main/java/com/cheatdatabase/listitems/CheatListItem.kt
package com.cheatdatabase.listitems

import com.cheatdatabase.data.model.Cheat

class CheatListItem : ListItem() {
    var cheat: Cheat? = null

    override fun type(): Int {
        return TYPE_CHEAT
    }

    override fun title(): String? {
        return cheat?.cheatTitle?.substring(0, 1)
    }
}